ConnectorRevit/ConnectorRevit/UI/ConnectorBindingsRevit.Receive.cs

```diff
@@ -277,9 +313,145 @@ private void DeleteObjects(IReceivedObjectIdMap<Base, Element> previousObjects,
         catch (Exception ex)
         {
           SpeckleLog.Logger.Warning(ex, "Failed to convert");
-          obj.Update(status: ApplicationObject.State.Failed, logItem: ex.Message);
+          obj.Log.Add($"{ex.Message}");
+
+          // reconvert as directShape if possible
+          if (shouldConvertAsDisplayable || isConvertibleAndDisplayable)
+          {
+            obj.Log.Add($"Retrying conversion as DirectShape fallback");
+            var convRes = ConvertAsDisplayable(@base, displayableSettings);
+            if (convRes != null)
+            {
+              obj.Update(
+                status: convRes.Status,
+                createdIds: convRes.CreatedIds,
+                converted: convRes.Converted,
+                log: convRes.Log
+              );
+            }
+            else
+            {
+              obj.Update(status: ApplicationObject.State.Failed, logItem: "Reconversion returned null");
+            }
+          }
           progress.Report.UpdateReportObject(obj);
         }
+
+        return obj;
+      }
+
+      // Retries a conversion using the direct mesh setting.
+      // Used in the case of failed conversions
+      ApplicationObject ConvertAsDisplayable(Base @base, Dictionary<string, string> displayableSettings)
+      {
+        converter.SetConverterSettings(displayableSettings);
```

Comment:
Not sure it's a good UX to change this on their behalf, I'd rather just pass that in the report instead.

Reply:
EDIT: let's add a new binding